include/sw_isr_table.h

```diff
@@ -65,8 +65,8 @@ struct _isr_list {
  * and/or SW ISR tables.
  */
 #define Z_ISR_DECLARE(irq, flags, func, param) \
-	static struct _isr_list Z_GENERIC_SECTION(.intList) __used \
-		_MK_ISR_NAME(func, __COUNTER__) = \
+	static struct _isr_list Z_GENERIC_SECTION(.intList) \
+	__aligned(sizeof(void *)) __used _MK_ISR_NAME(func, __COUNTER__) = \
```

Comment:
What was going wrong here?  By my read struct _isr_list includes a void* field, so should naturally have had pointer alignment already.

I worry that this might be working around a deeper problem with the linker scripts...